src/transformers/models/transfo_xl/modeling_transfo_xl.py

```diff
@@ -673,8 +673,8 @@ class TransfoXLLMHeadModelOutput(ModelOutput):
     Base class for model's outputs that may also contain a past key/values (to speed up sequential decoding).
 
     Args:
-        losses (:obj:`torch.FloatTensor` of shape `(batch_size, sequence_length-1)`, `optional`, returned when ``labels`` is provided)
```

Comment:
This is a pretty strong backward breaking change. Could we maybe instead try to just alias to losses with a `@property` function, e.g.:

```python
@property
def loss(self):
    return self.losses
```